workflows/gone.py

```diff
@@ -61,6 +62,14 @@ def ts2plink(ts,ped_file,map_file,gm_chr):
 
     # make .ped file
     genomat = ts.genotype_matrix() + 1 # it takes alleles (1,2)
+    # mask if appropriate
+    if type(mask_intervals) is not None:
+        positions = ts.tables.sites.position
```

Comment:
https://tskit.dev/tskit/docs/latest/python-api.html#tskit.TreeSequence.sites_position
```suggestion
        positions = ts.sites_position
```